package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "global-tunnel-ng",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
Edited my pull request... hope that's sufficient. I'm very new to github and hope not to mess around to much.